jericho/jericho.py

```diff
@@ -369,36 +370,40 @@ def __init__(self, story_file, seed=-1):
         self.seed = seed
         self.frotz_lib.setup(self.story_file, self.seed)
         self.player_obj_num = self.frotz_lib.get_self_object_num()
+        try:
+            self.bindings = _load_bindings(story_file)
```

Comment:
Should `_load_bindings` rely on the story hash rather than the filename?

Reply:
Yes, good idea!